week-1/L-functions-nested/exercise2.js

```diff
@@ -0,0 +1,5 @@
+var mentor1 = "Daniel";
+var mentor2 = "Irina";
+var mentor3 = "Mimi";
+var mentor4 = "Rob";
+var mentor5 = "Yohannes";
```

Comment:
you need to write a program that logs a shouty greeting to each one.
* Your program should include a function that spells their name in uppercase, and a function that creates a shouty greeting.
---> 
HELLO DANIEL
HELLO IRINA
HELLO MIMI
HELLO ROB
HELLO YOHANNES